emitter/textlog/emitter.go

```diff
@@ -64,6 +65,18 @@ func Emitter(w io.Writer, opt ...Option) alog.Emitter {
 			}
 			m.WriteString("] ")
 		}
+		if t := e.STags; len(t) != 0 {
+			m.WriteByte('[')
+			for i, p := range t {
+				if i != 0 {
+					m.WriteByte(' ')
+				}
+				m.WriteString(p.Key)
+				m.WriteByte('=')
+				m.WriteString(fmt.Sprintf("%+v", p.Val))
```

Comment:
It would probably be better to use `fmt.Fprintf(m, "%+v", p.Val)` here.
(it will often boil down to something equivalent, but it can also avoid some extra allocations)